deploy/role.yaml

```diff
@@ -82,3 +82,15 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - "policy.mcm.ibm.com"
+  resources:
+  - policies
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Do we need all these verbs for this api group?

Reply:
Yep, al except patch. I think I could get rid of that

Reply:
Done